cromshell

```diff
@@ -1386,21 +1386,31 @@ function notify()
     [[ $? -ne 0 ]] && error "ERROR: Could not copy cromshell to server ${hostServer}" && error "$(cat ${tmpOut})" && exit 7
 
     # Spin off notification process on the server:
-    results=$( ssh ${hostServer} "~/${SCRIPTNAME} notify ${WORKFLOW_ID} ${email} ${WORKFLOW_SERVER_URL}" )
+    results=$( ssh ${hostServer} "~/${SCRIPTNAME} _rawNotify ${WORKFLOW_ID} ${email} ${WORKFLOW_SERVER_URL}" )
     [[ $? -ne 0 ]] && error "ERROR: Could not start notification daemon on ${hostServer}" && exit 8
 
     # Let the user know we've done our job:
     echo "Spun off thread on ${hostServer} - PID = $( echo "${results}" | grep "Spun off thread on PID" | sed 's#Spun off thread on PID ##g' )"
   else
-    error "Spinning off notification to ${email} thread for"
-    error "    workflow:             ${WORKFLOW_ID}" 
-    error "    from Cromwell server: ${WORKFLOW_SERVER_URL}"
-    error "..."
-    nohup bash -c "source ${SCRIPTDIR}/${SCRIPTNAME}; _notifyHelper ${WORKFLOW_ID} ${email} ${WORKFLOW_SERVER_URL}" &>/dev/null &
-    echo "Spun off thread on PID $!"
+		_rawNotify ${WORKFLOW_ID} ${email} ${WORKFLOW_SERVER_URL}
   fi
 }
 
+# Perform a raw notification without validating any input arguments.
+function _rawNotify() 
+{
+	local WORKFLOW_ID=$1
+	local email=$2
+	local WORKFLOW_SERVER_URL=$3
+
+	error "Spinning off notification thread to ${email} for"
+	error "    workflow:             ${WORKFLOW_ID}" 
+	error "    from Cromwell server: ${WORKFLOW_SERVER_URL}"
+	error "..."
+	nohup bash -c "source ${SCRIPTDIR}/${SCRIPTNAME}; _notifyHelper ${WORKFLOW_ID} ${email} ${WORKFLOW_SERVER_URL}" &>/dev/null &
+	echo "Spun off thread on PID $!"
```

Comment:
Should this also be error?

Reply:
Yup.  Good catch!

Reply:
err... Ideally both it and a previous message would be errors, but making them errors causes messages to be parsed incorrectly.  

I'll file a separate issue for it (#97)